server/routes/dashboardCollectionOverview.tsx

```diff
@@ -38,20 +44,27 @@ app.get('/dash/collection/:collectionSlug/overview', async (req, res, next) => {
 		}
 
 		const overviewData = await getCollectionOverview(initialData);
+		const submissionWorkflowPresent = await Promise.all([
+			getSubmissionWorkflow(overviewData.collection.id),
+		]);
```

Comment:
With the above proposal we could change this statement to:

```ts
const hasSubmissionWorkflow = await collectionHasSubmissionWorkflow(overviewData.collection.id);
```

(and remove the below `submissionWorkflows = submissionWorkflowPresent[0];`)

Reply:
Otherwise, you can drop the `Promise.all` here:

```ts
const submissionWorkflowPresent = await getSubmissionWorkflow(overviewData.collection.id);
```